CONTRIBUTING.md

```diff
@@ -1,8 +1,75 @@
-## Guide
+# Contributing to GNO
 
-### Getting started with ViM
+Thank you for looking to contribute to the GNO project.
+We appreciate every open-source contribution, as it helps us improve and enhance gno for the benefit of the community.
 
-Add to your .vimrc file:
+This document outlines some basic pointers on making your future contribution a great experience. It outlines basic PR
+etiquette employed by the core gno team. It lays out coding styles, simple how-to guides and tools to get you up and
+running and pushing code.
+
+If you are unsure about something, please don’t hesitate to reach out for help by opening an issue or discussion.
```

Comment:
```suggestion
If you are unsure about something, please don’t hesitate to reach out for help by opening an issue here or discuss on Discord.
```

What do you think about disabling discussions and only use issues instead?

Reply:
You're right, we don't have GitHub discussions enabled yet 👍 

Updated:

[04881f2](https://github.com/gnolang/gno/pull/469/commits/04881f263035f1d561ff73d20002a71cafdeb099)